sdk/eventgrid/azure-eventgrid/README.md

```diff
@@ -76,7 +74,7 @@ An event [domain](https://docs.microsoft.com/azure/event-grid/event-domains) is
 When you create an event domain, a publishing endpoint for this domain is made available to you. This process is similar to creating an Event Grid Topic. The only difference is that, when publishing to a domain, you must specify the topic within the domain that you'd like the event to be delivered to.
 
 ### Event schemas
-An [**event**](https://docs.microsoft.com/azure/event-grid/concepts#events) is the smallest amount of information that fully describes something that happened in the system. When a custom topic or domain is created, you must specify the schema that will be used when publishing events.
+An **[event](https://docs.microsoft.com/azure/event-grid/concepts#events)** is the smallest amount of information that fully describes something that happened in the system. When a custom topic or domain is created, you must specify the schema that will be used when publishing events.
```

Comment:
nit: I see we have `**` here for the word event, but not for the above `domain` or `event`. either have `**` on all of them of remove it for consistency :P

Reply:
added ** everywhere